be/src/util/runtime_profile.h

```diff
@@ -491,9 +492,9 @@ class RuntimeProfile {
     /// All counters in this profile must be of unit AveragedCounter.
     bool _is_averaged_profile;
 
-    // Map from counter names to counters.  The profile owns the memory for the
-    // counters.
-    typedef std::map<std::string, Counter*> CounterMap;
+    // Map from counter names to counters and parent counter names.
+    // The profile owns the memory for the counters.
+    typedef std::map<std::string, std::pair<Counter*, std::string>> CounterMap;
```

Comment:
Why need this change?

Reply:
For profile prune. `RuntimeProfile::remove_counters` will remove all the counters except the `saved_counter_names`, and we need find all it's parent counters of those `saved_counter_names`